### ISSUE TITLE
ImportError: cannot import name 'Protocol' from 'typing' (python 3.7.11)

### ISSUE THREAD

**Issue by karwojan** [MGHComputationalPathology/dicomweb-client#58]
In new conda environment with only dicomweb-client 0.54.2 installed, when using python 3.7.11, following error has occured:
```
>>> import dicomweb_client.api
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/jan/.conda/envs/test/lib/python3.7/site-packages/dicomweb_client/__init__.py", line 3, in <module>
    from dicomweb_client.api import DICOMwebClient, DICOMfileClient
  File "/home/jan/.conda/envs/test/lib/python3.7/site-packages/dicomweb_client/api.py", line 4, in <module>
    from dicomweb_client.protocol import DICOMClient
  File "/home/jan/.conda/envs/test/lib/python3.7/site-packages/dicomweb_client/protocol.py", line 1, in <module>
    from typing import (
ImportError: cannot import name 'Protocol' from 'typing' (/home/jan/.conda/envs/test/lib/python3.7/typing.py)
```
I guess, that typing-extensions is not used in case of python versions lower then 3.8.

**Comment by hackermd** [comment]
Thanks for reporting the issue @karwojan.

It should be installed, since the dependency is flagged with [python_version](https://setuptools.pypa.io/en/latest/userguide/dependency_management.html#platform-specific-dependencies) (see [setup.py](https://github.com/MGHComputationalPathology/dicomweb-client/blob/d1d82d45bd42ab2c4a15436365e82d6b6432cdb0/setup.py#L58)). Maybe an issue with conda?

**Comment by hackermd** [comment]
Good point @ntenenz!

@karwojan I started to put together a pull request https://github.com/MGHComputationalPathology/dicomweb-client/pull/59, but haven't tested it yet.

**Comment by karwojan** [comment]
Thanks a lot @hackermd , I think it should definitely fix the issue. I'm waiting for a new version release.

**Comment by hackermd** [comment]
@karwojan should be fixed with v0.54.3: https://pypi.org/project/dicomweb-client/0.54.3
